assets/js/components/sidebar/sidebar.js

```diff
@@ -1,11 +1,14 @@
-import { ComponentHelper } from '../component-helper.js'
+import { ComponentHelper } from '../component-helper.js';
 
 export class Sidebar extends HTMLElement {
-	constructor() {
```

Comment:
In this file you've made lots of changes to the formatting but only one actual necessary change. It would have been better to break these changes into multiple commits because it makes it easier for the reviewer to immediately see what exactly you've changed. Nothing to change, just something to note for the future.